### ISSUE TITLE
Mist toggles to Testnet when hitting AltGr+2

### ISSUE THREAD

**Issue by mpolavieja** [ethereum/mist#379]
I am running Mist 0.5.2 on Windows 7 Home Premium 64bits

When I try to key the character "@", which in my spanish keboard is located at "AltGr +2", Mist toggles to Testnet mode!!!!  instead of just writing "@".

**Comment by alexvandesande** [comment]
Is this a common keystroke shortcut for such a common key? Most keyboards just use shift and alt for characters, not Control, which is why we picked control + alt + 2

**Comment by mpolavieja** [comment]
Hi Alexandre,

Thank you for your swift response.   Mist closes only when I hit "AltGr +
2", I did not hit the Control key.

For the standard spanish keyboard there are quite a a few characters like
"[", "]","{","}", "\", "|", "@", "#", "~", "¬" which need to combine with
"AltGr" in order to be typed (the "Alt" key does not work for this in the
spanish keyboard, we must press "AltGr").   See the image below, for the
keys that have 3 characters, the one at the downright corner needs a
combination with AltGr in order to be stroke.



[image: Imágenes integradas 1]

Thank you,

Manuel Polavieja


2016-03-17 15:06 GMT+01:00 Alexandre Van de Sande <notifications@github.com>
:

**Comment by BenTuras** [comment]
You could try ALT(hold down for all 3 digits) 064(on numeric keyboard) to get an @

**Comment by mpolavieja** [comment]
Oh, my workaraund was just to copy and paste the characther "@" from the
notepad. It is nice to know that other workaround though.  Thanks!

But please take in consideration that it can be quite ackward for spanish
standard user that Mist just closes when trying to type "@".

Moreover, is it suposed to close when hitting "Ctrl+Alt+2", or when hitting
"AltGr+2".    Becuase in my case it is closing when hitting "AltGr+2".

Regards,
Manuel.

2016-03-18 9:16 GMT+01:00 Ben <notifications@github.com>:

**Comment by alexvandesande** [comment]
Thanks Manuel. I had never heard of AltGr before and I don't know why electron translates alt+command to it. 

Well look for other alternatives

**Comment by mpolavieja** [comment]
Thanks Alex. 

I did not explain well in my last email.    I wrongly said that Mist closed when trying to hit "@", But what happens Is that it switches to tesnet, as I explained in my first email. Sorry. 

I believe the AltGr key exists only in spanish keyboards.  The one at the left of the spacebar is regular "Alt" and the one at the right is "AltGr" which is used to access quite a few symbols. 

Regards,

Manuel.
